pkg/apis/serving/v1beta1/configuration_validation.go

```diff
@@ -61,3 +64,26 @@ func (cs *ConfigurationStatus) Validate(ctx context.Context) *apis.FieldError {
 func (csf *ConfigurationStatusFields) Validate(ctx context.Context) *apis.FieldError {
 	return nil
 }
+
+// ValidateLabels function validates service labels
+func (c *Configuration) ValidateLabels() *apis.FieldError {
+	var errs *apis.FieldError
+LabelLoop:
+	for key, val := range c.GetLabels() {
+		switch {
+		case key == config.VisibilityLabelKey:
+			errs = errs.Also(validateClusterVisbilityLabel(val))
+		case key == serving.RouteLabelKey:
```

Comment:
Do you want to specifically exclude RouteLabelKey? Go does not fallthrough unless you tell it to.

Reply:
Yes I want to exclude `routeKey` specifically because that key will be true for case ` strings.HasPrefix(key, serving.GroupNamePrefix)`  and it throw an error.